src/main/java/com/firebolt/jdbc/type/BaseType.java

```diff
@@ -147,6 +147,13 @@ private <T> T transform(StringToColumnTypeConversion conversion) throws SQLExcep
 		return (T) transformFunction.apply(conversion);
 	}
 
+	private static String checkInfinity(String s) {
+		if ("inf".equals(s) || "-inf".equals(s)) {
```

Comment:
Let's create const/enum (whichever is java's best practice) for those inf values. This way it will be easier to use them elsewhere if needed.

Reply:
Good comment, but I did something else: reused already existing functions